### ISSUE TITLE
How to get the result of the validation set during the train set

### ISSUE THREAD

**Issue by dongzhenguo2016** [TuSimple/simpledet#325]
I have a question!
How to calculate the classification accuracy and positioning accuracy of the current model on the validation set for every epoch in the training process, just like YOLOv3 ?
Thanks a lot

**Comment by tiberium24** [comment]
I also face the same issue. At least I want to plot the validation loss somehow.

**Comment by liuhangfan** [comment]
same issue
